src/com/group6/users/UserStorage.java

```diff
@@ -0,0 +1,51 @@
+import java.io.*;
+import java.util.HashMap;
+import java.util.Map;
+
+public class UserStorage {
+    public static Map<String, User> loadUsers(){

```

Comment:
I like it!

But consider making this into a normal class, then change out the static methods to non static to keep it "OO". 

Could have a field called `applicantsFilepath`, `officersFilepath`, and `managersFilepath` so you dont hardcode the loading within the method, rather you use the instance fields.